docs/guides/component-testing/overview.mdx

```diff
@@ -1,7 +1,14 @@
 ---
-title: Cypress Component Testing
+title: Component Testing Cypress | Browser-Based Testing | [Learn How]
 sidebar_position: 10
 sidebar_label: Overview
+description:
+  Quickly build and test components with multiple front-end UI libraries — no
+  matter how simple or complex - with Cypress Component Testing. Learn how.
+---
+
+# Cypress Component Testing
```

Comment:
Since we want a different Meta Title (line 2), I had to add this H1 to the markdown. I added a horizontal rule after the H1, but the line's styling is a bit different compared to other pages. Not a big deal, but I just wanted to call it out.